point_angle_range_to_polygon.py

```diff
@@ -12,47 +12,67 @@ def createPointAngleRangePolygons(points, outputPolygons, azimuthField, rangeFie
     azimuthField = azimuthField
     rangeField = rangeField
     beamWidthField = beamWidthField
-    #Polygon feature class will created by the tool. 
     outputPolygonFeatures = outputPolygons
     polygons = []
-    
-    cursorFields = ["SHAPE@", azimuthField, rangeField, beamWidthField]
+    #Setup field list for point search cursor and polygon insert cursor    
+    cursorFields = [f.name for f in arcpy.ListFields(pointFeatures)]
```

Comment:
this is a nice use of list comprehension. fwiw you could add an if statement to the comprehension to filter out shape. but what you have is explicit and readable.